misk-hibernate/src/main/kotlin/misk/vitess/StartVitessService.kt

```diff
@@ -269,14 +319,25 @@ class StartVitessService(val config: DataSourceConfig) : AbstractIdleService() {
     val docker: DockerClient = DockerClientBuilder.getInstance()
         .withDockerCmdExecFactory(NettyDockerCmdExecFactory())
         .build()
+    /**
+     * Vitess clusters are cached statically so we create a static instance of ResourceLoaderModule.
```

Comment:
ya want this new thing: `ResourceLoader.SYSTEM`

Reply:
NICE!